src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/ProjectView.xaml.cs

```diff
@@ -161,15 +161,26 @@ protected override void OnKeyUp(KeyEventArgs e)
                         if (_versions.SelectedIndex >= 0 && isAVersion)
                         {
                             // Check if the user typed a custom version
-                            if (isUserInputValidNuGetVersionRange)
+                            if (isUserInputValidNuGetVersionRange && userRequestedVersionRange != null && _versions.Items?.CurrentItem != null)
```

Comment:
This `if` block is nested inside another if block that executes only when `isAVersion` is true. For `isAVersion` to be `true` then `_versions.Items.CurrentItem != null`. Therefore, the same condition is not required to be evaluated in the nested if block.